src/modules-post.js

```diff
@@ -0,0 +1,10 @@
+Object.assign(ClassHandle.prototype, {
+  'preventAutoDelete' () {
+    const index = deletionQueue.indexOf(this);
+    if (index > -1) {
+      deletionQueue.splice(index, 1);
+    }
+    this.$$.deleteScheduled = false;
+    return this;
+  }
+});
```

Comment:
Using `--post-js` seemed easier to maintain instead of patching `embind.js` directly and updating the patch on each new release of emscripten https://github.com/emscripten-core/emscripten/blob/a1fe3902bf73a3802eae0357d273d0e37ea79898/src/embind/embind.js#L1572